open-api/rest-catalog-open-api.yaml

```diff
@@ -3809,6 +4150,41 @@ components:
             }
           }
 
+    #  Note that this is a representative example response for use as a shorthand in the spec.
+    #  The fields `message` and `type` as indicated here are not presently prescriptive.
+    MisdirectedRequestResponse:
+      description:
+        Misdirected Request. The request was directed to a server that is not able to produce a response.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/IcebergErrorResponse'
+          example: {
+            "error": {
+              "message": "Fail to plan table scan: must perform preplan before planning a table scan",
+              "type": "PreplanTableRequiredException",
+              "code": 421
+            }
+          }
+
+    #  Note that this is a representative example response for use as a shorthand in the spec.
+    #  The fields `message` and `type` as indicated here are not presently prescriptive.
```

Comment:
Is it necessary to copy this comment for every case? I thought that this was already clear anyway.

Reply:
@rdblue 
Not sure why this was copied over for some of the error response objects. I see this [here](https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml#L3724) for three of the objects `ForbiddenResponse, UnsupportedOperationResponse, UnauthorizedResponse`. I can remove this comment from the new error response objects introduced in this pr.

I think @jackye1995 may have assumed we require this at the top of each new error response object we introduce.